org.snesrev.Zelda3.yml

```diff
@@ -0,0 +1,64 @@
+app-id: org.snesrev.Zelda3
```

Comment:
`snesrev.org` does not exist.

Reply:
Would be `com.github.snesrev.Zelda3` okay?

Reply:
`io.github.snesrev.Zelda3` as per https://docs.flatpak.org/en/latest/conventions.html#application-ids